tests/recovery.py

```diff
@@ -499,7 +503,7 @@ def run(args):
                         ccf.ledger.WELL_KNOWN_SINGLETON_TABLE_KEY
                     ]
                 )["status"]
-                if service_status == "Opening":
+                if service_status == "Opening" or service_status == "Recovering":
                     LOG.info(f"New ledger chunk found for service opening at {seqno}")
```

Comment:
This line and the one three down could be updated to say Opening/Recovering as appropriate.